x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_modal/datafeed_modal.tsx

```diff
@@ -0,0 +1,350 @@
+              popoverPlacement="left-start"
+            />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiModalHeader>
+
+      <EuiModalBody>
+        <EuiTabs size="s">{renderTabs()}</EuiTabs>
+        <EuiSpacer size="m" />
+        {isLoadingChartData && <EuiLoadingChart size="l" />}
+        {!isLoadingChartData && selectedTabId === TAB_IDS.CHART && (
+          <EuiFlexGroup direction="column">
+            <EuiFlexItem grow={false}>
+              <EuiFlexGroup justifyContent="spaceBetween">
+                <EuiFlexItem grow={false}>
+                  <EuiSelect
+                    options={getIntervalOptions(bucketSpan)}
+                    value={interval}
+                    onChange={(e) => setInterval(e.target.value)}
+                    aria-label={i18n.translate(
+                      'xpack.ml.jobsList.datafeedModal.intervalSelection',
+                      {
+                        defaultMessage: 'Datafeed modal chart interval selection',
+                      }
+                    )}
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EditQueryDelay datafeedConfig={datafeedConfig} />
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiFlexGroup gutterSize="none" alignItems="center">
+                <EuiFlexItem grow={false}>
+                  <EuiButtonEmpty
+                    color="primary"
+                    size="l"
+                    onClick={() => {
+                      handleEndDateChange(CHART_DIRECTION.BACK);
+                    }}
+                    iconType="arrowLeft"
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <Chart size={CHART_SIZE}>
+                    <Settings showLegend showLegendExtra legendPosition={Position.Bottom} />
+                    <Axis
+                      id="bottom"
+                      position={Position.Bottom}
+                      showOverlappingTicks
+                      tickFormat={dateFormatter}
+                      title={i18n.translate('xpack.ml.jobsList.datafeedModal.xAxisTitle', {
+                        defaultMessage: 'Bucket span ({bucketSpan})',
+                        values: { bucketSpan },
+                      })}
+                    />
+                    <Axis
+                      id="left"
+                      title={i18n.translate('xpack.ml.jobsList.datafeedModal.yAxisTitle', {
+                        defaultMessage: 'Record count',
+                      })}
+                      position={Position.Left}
+                    />
+                    <LineSeries
+                      color={euiTheme.euiColorPrimary}
+                      id={i18n.translate('xpack.ml.jobsList.datafeedModal.sourceSeriesId', {
+                        defaultMessage: 'Source doc count by bucket span',
```

Comment:
IMO this label could be simplified to avoid duplicating the chart axis labels. For example:
```suggestion
                        defaultMessage: 'Source indices',
```

Reply:
Updated in b201a635f08667349e20b1631023c46d1162a3b5